### PR TITLE
#186: Changing the design of my location button to correspond to the …

### PR DESCRIPTION
**Pull request by Hambrsoom**
I redesigned the current location button to look like the one in our mockups as shown in the below figure. 
image

### CODE REVIEW

**Comment by ferdousara-parvin** [comment]
Can the background of the button be white and not grey pls?

File: app/src/main/res/drawable/round_button_gray.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_selected="true">
+        <shape android:shape="oval">
+            <solid android:color="@color/backgroundGrey"/>
```
Comment (ferdousara-parvin) [inline app/src/main/res/drawable/round_button_gray.xml:5]: white pls

File: app/src/main/res/drawable/round_button_gray.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+            <size android:height="25dp"
+                android:width="25dp"/>
+        </shape>
+    </item>
+    <item android:state_selected="false">
+        <shape android:shape="oval">
+            <solid android:color="@color/backgroundGrey"/>
```
Comment (ferdousara-parvin) [inline app/src/main/res/drawable/round_button_gray.xml:12]: white pls

File: app/src/main/res/drawable/round_button_gray.xml (hunk 1 above)
Comment (Hambrsoom) [reply app/src/main/res/drawable/round_button_gray.xml:5]: sure not a problem :)

**Pull request merged**